Uchu.StandardScripts/NinjagoMonastery/GoToNpcBodge.cs

```diff
@@ -0,0 +1,51 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.NinjagoMonastery
+{
+    [ZoneSpecific(2000)]
+    public class GoToNpcBodge : NativeScript
```

Comment:
Changing this to a migrate for the CDClient was discussed before after this script was created. Was this looked into?

Reply:
It was not, no. It might also be possible to get rid of this script by changing how GoToNpc tasks are handled (see also the conversation on Discord about this), but this will need to be investigated more (by eg. seeing what happens when the tasks are always regarded as complete, or if the value of target_objectID could be used)

Reply:
Probably should wait to get some type of automation testing going on this or for this to be a problem. Like other changes to the mission system, I changing the behavior leading to unrelated problems elsewhere.